easybuild/easyconfigs/a/ANTLR/ANTLR-2.7.7-intel-2015a.eb

```diff
@@ -0,0 +1,31 @@
+easyblock = 'ConfigureMake'
+
+name = 'ANTLR'
+version = "2.7.7"
+
+homepage = 'http://www.antlr2.org/'
+description = """ANTLR, ANother Tool for Language Recognition, (formerly PCCTS)
+ is a language tool that provides a framework for constructing recognizers,
+ compilers, and translators from grammatical descriptions containing
+ Java, C#, C++, or Python actions."""
+
+toolchain = {'name': 'intel', 'version': '2015a'}
+  
+source_urls = ['http://www.antlr2.org/download/']
+sources = [SOURCELOWER_TAR_GZ]
+
+patches = ['%(name)s-%(version)s_includes.patch']
+
+dependencies = [
+    ('Java', '1.8.0_31', '', True),
+    ('Python', '2.7.9'),
```

Comment:
if Python is a dep, we usually reflect that via a versionsuffix...

also: 2.7.10?